articles/TOC.yml

```diff
@@ -1,44 +1,51 @@
 - name: Microsoft Quantum Documentation
   href: index.yml
-- name: Overview
+- name: Getting started
   items:
-  - name: Get started
-    href: welcome.md
-  - name: Five questions about quantum computing
-    items:
-    - name: What is quantum computing?
-      href: overview/what-is-quantum-computing.md
-    - name: What can a quantum computer do?
-      href: overview/quantum-computers.md
-    - name: Why learn quantum computing?
-      href: overview/why-learn-quantum-computing.md
-    - name: What is Q#?
-      href: overview/what-is-qsharp.md
-    - name: How to learn quantum computing?
-      href: overview/how-to-learn-quantum-computing.md
-- name: Install and update
+  - name: What is quantum computing?
+    href: overview/what-is-quantum-computing.md
+  - name: What can a quantum computer do?
+    href: overview/quantum-computers.md
+  - name: Why learn quantum computing?
+    href: overview/why-learn-quantum-computing.md
+  - name: What are Q# and QDK?
+    href: overview/what-is-qsharp.md
+  - name: How can I learn more?
+    href: overview/how-to-learn-quantum-computing.md
+- name: The QDK introduction
+  href: welcome.md
   items:
-  - name: Install the QDK
+  - name: Install and update
     items:
-    - name: Installation overview
-      href: install-guide/index.md
-    - name: Q# + C#
-      href: install-guide/csinstall.md
-    - name: Q# + Python
-      href: install-guide/pyinstall.md
-    - name: Q# Jupyter notebooks
-      href: install-guide/qjupyter.md
-  - name: Update the QDK
-    href: install-guide/update.md  
-- name: Quickstarts
+    - name: Install the QDK
+      items:
+      - name: Installation overview
+        href: install-guide/index.md
+      - name: Q# + C#
+        href: install-guide/csinstall.md
+      - name: Q# + Python
+        href: install-guide/pyinstall.md
+      - name: Q# Jupyter notebooks
+        href: install-guide/qjupyter.md
+    - name: Update the QDK
+      href: install-guide/update.md
+    - name: How to create a Q# project
+      href: how-to-guides/create-qsharp-project.md
+- name: Q# examples
   items:
-  - name: Quantum random number generator
-    href: quickstarts/qrng.md
-  - name: Quantum basics with Q#
-    href: quickstart.md
-  - name: Implement Grover's search algorithm
-    href: quickstarts/search.md
-- name: Concepts
+  - name: Quickstarts
+    items:
+    - name: Quantum basics with Q#
+      href: quickstart.md
+    - name: Quantum random number generator
+      href: quickstarts/qrng.md
+    - name: Implement Grover's search algorithm
+      href: quickstarts/search.md
+  - name: Learn Q# with the Quantum Katas
+    href: intro-to-katas.md
+  - name: Sample code
+    href: https://docs.microsoft.com/en-us/samples/browse/?languages=qsharp
```

Comment:
Same here with the indentation. People may think "Learn Q# with Quantum Katas" and "Code samples" are part of the "Quickstart". Can we align these two with the arrow? 

![image](https://user-images.githubusercontent.com/59890648/73984863-8f312380-48ee-11ea-9882-bb572344b497.png)